packages/app-scm/git-cola/files/prefix-hacks.patch

```diff
@@ -0,0 +1,38 @@
+Upstream: no
+Reason: assumes being installed in /usr/bin and uses sick hacks to locate it's data.
+
+diff --git a/bin/git-cola b/bin/git-cola
+index 91eab7ec..6dc945c3 100755
+--- a/bin/git-cola
++++ b/bin/git-cola
+@@ -33,10 +33,10 @@ def setup_environment():
+     # Look for modules in the source or install trees
+     if python2:
+         cola_mod = join(prefix, str('cola'), str('__init__.py'))
+-        install_lib = join(prefix, str('share'), str('git-cola'), str('lib'))
++        install_lib = join(dirname(prefix), str('share'), str('git-cola'), str('lib'))
```

Comment:
Upstream really should install python libraries into site-packages instead of messing with sys.path. Your idea sounds better, but I suggest to keep this patch downstream until somebody will be willing to make things right.